cmd/apidocs/create.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package apidocs
+
+import (
+	"fmt"
+	"strconv"
+
+	"internal/apiclient"
+	"internal/client/apidocs"
+
+	"github.com/spf13/cobra"
+)
+
+// CreateCmd to create a catalog items
+var CreateCmd = &cobra.Command{
+	Use:   "create",
+	Short: "Create a new catalog item",
+	Long:  "Create a new catalog item",
+	Args: func(cmd *cobra.Command, args []string) (err error) {
+		_, err = strconv.ParseBool(published)
+		if err != nil {
+			return fmt.Errorf("published must be a boolean value: %v", err)
+		}
+		_, err = strconv.ParseBool(anonAllowed)
+		if err != nil {
+			return fmt.Errorf("allow-anon must be a boolean value: %v", err)
+		}
+
+		_, err = strconv.ParseBool(requireCallbackUrl)
+		if err != nil {
+			return fmt.Errorf("require-callback-url must be a boolean value: %v", err)
+		}
+
+		return apiclient.SetApigeeOrg(org)
+	},
+	RunE: func(cmd *cobra.Command, args []string) (err error) {
+		_, err = apidocs.Create(siteid, title, description, published,
+			anonAllowed, apiProductName, requireCallbackUrl, imageUrl, categoryIds)
+		return
+	},
+}
+
+var (
+	title, description, published, anonAllowed, apiProductName string
+	requireCallbackUrl, imageUrl                               string
+	categoryIds                                                []string
+)
+
+func init() {
+	CreateCmd.Flags().StringVarP(&title, "title", "t",
```

Comment:
Conflicts with -t for token, I'm guessing

gorun apidocs create --help
panic: unable to redefine 't' shorthand in "create" flagset: it's already used for "title" flag